generators/aws-swarm-tokens/templates/aws-swarm-tokens/index.ts

```diff
@@ -0,0 +1,51 @@
+import { remote } from "@pulumi/command";
+import { secret } from "@pulumi/pulumi";
+
+import { getConfig } from "./config";
+
+export = async () => {
+
+  const config = await getConfig();
+
+  if (!config.leaderIp) {
```

Comment:
rename `leaderIp` to `managerIp`. There is no reason why we couldn't run this project on a swarm with multiple managers